docs/pages/machine-id/reference/configuration.mdx

```diff
@@ -46,58 +47,106 @@ storage:
+    # Directory specifies a filesystem directory where short-lived certificates
+    # are stored.
     - directory:
-        # Configure the path at which to store certificates and other
-        # artifacts.
-        path: /opt/machine-id
-
-        # Configure symlink attack prevention. Requires Linux 5.6+.
-        # Possible values:
-        #   * try-secure (default): Attempt to securely read and write certificates
-        #     without symlinks, but fall back (with a warning) to insecure read
-        #     and write if the host doesn't support this.
-        #   * secure: Attempt to securely read and write certificates, with a hard error
-        #     if unsupported.
-        #   * insecure: Quietly allow symlinks in paths.
-        symlinks: try-secure
-        
-        # Configure ACL use. Requires Linux with a file system that supports ACLs.
-        # Possible values:
-        #   * try (default on Linux): Attempt to use ACLs, warn at runtime if ACLs
-        #     are configured but invalid.
-        #   * off (default on non-Linux): Do not attempt to use ACLs.
-        #   * required: Always use ACLs, produce a hard error at runtime if ACLs
-        #     are invalid.
-        acls: try
+          # Configure the path at which to store certificates and other
+          # artifacts.
+          path: /opt/machine-id
+
+          # Configure symlink attack prevention. Requires Linux 5.6+.
+          # Possible values:
+          #   * try-secure (default): Attempt to securely read and write certificates
+          #     without symlinks, but fall back (with a warning) to insecure read
+          #     and write if the host doesn't support this.
+          #   * secure: Attempt to securely read and write certificates, with a hard error
+          #     if unsupported.
+          #   * insecure: Quietly allow symlinks in paths.
+          symlinks: try-secure
         
-        # One of more roles to grant to the bot. It must have been granted (at
-        # least) these roles with `tctl bots add --roles=...`
-        # By default, all possible roles are included.
+          # Configure ACL use. Requires Linux with a file system that supports ACLs.
+          # Possible values:
+          #   * try (default on Linux): Attempt to use ACLs, warn at runtime if ACLs
+          #     are configured but invalid.
+          #   * off (default on non-Linux): Do not attempt to use ACLs.
+          #   * required: Always use ACLs, produce a hard error at runtime if ACLs
+          #     are invalid.
+          acls: try
         
-        # Subset of roles allowed during creation via `tctl bots add --roles=...`. Can
-        # be used to write short-lived certificates with different roles to
-        # different directories.
-        roles: [a, b, c]
+      # One of more roles to grant to the bot. It must have been granted (at
+      # least) these roles with `tctl bots add --roles=...`
```

Comment:
Maybe use a different word for the second instance of "grant"? A reader might ask, if these roles have already been granted to the bot via `tctl bots add`, why we would also need to grant them here.